docs/src/components/pricing/PricingTable.tsx

```diff
@@ -636,6 +663,16 @@ const rowHeaders: Record<string, React.ReactNode> = {
 const yes = <IconImage name="pricing/yes" title="Included" />;
 const pending = <IconImage name="pricing/time" title="Work in progress" />;
 const no = <IconImage name="pricing/no" title="Not included" />;
+const toBeDefined = (
+  <Typography
+    component="a"
+    href="https://forms.gle/19vN87eBvmXPjBVp6"
+    sx={{ display: 'inline-block' }}
```

Comment:
The style of this link doesn't match the rest of the table very well. I'm suggesting a smaller font here, but I think the colors don't seem to match the theme either.

(Also, just noticed that we have many different font sizes on the same column. 
We should probably fix that– in another PR)

```suggestion
    sx={{ display: 'inline-block', fontWeight: '0.875em' }}
```

Reply:
I copied how links are made for the first column